src/frameworks/angular/index.ts

```diff
@@ -238,10 +239,11 @@ exports.handle = function(req,res) {
   });
 };\n`;
   } else if (serverOutputPath) {
+    const serverOutputPosixPath = serverOutputPath.split(sep).join(posix.sep);
```

Comment:
I'd suggest setting the values of the correct paths directly in the `getServerConfig` function.
We can adjust both: `serverOutputPath` and `browserOutputPath`.

What do you think?

Reply:
Yes, I do like this idea 👍

Reply:
I have changed it, to correct the paths directly in the `getServerConfig` function.